app/controllers/invitations_controller.rb

```diff
@@ -3,13 +3,17 @@ class InvitationsController < ApplicationController
   before_action :set_invitation, :verify_invitation_validity, only: [:redirect]
   skip_before_action :authenticate_agent!, only: [:invitation_code, :redirect, :redirect_shortcut]
 
-  def create
+  def create # rubocop:disable Metrics/AbcSize
     if invite_user.success?
       respond_to do |format|
         format.json { render json: { success: true, invitation: invitation } }
         format.pdf { send_data pdf, filename: pdf_filename, layout: "application/pdf" }
+        format.turbo_stream { redirect_to structure_user_follow_ups_path(@user.id) }
       end
     else
+      return turbo_stream_display_error_modal(invite_user.errors) if request.format.turbo_stream?
+
+      # we want to render json when the request is json but also when the request is pdf
       render json: { success: false, errors: invite_user.errors }, status: :unprocessable_entity
```

Comment:
on peut faire quelque chose comme ça pour garder les conventions et se passer du commentaire
```suggestion 
respond_to do |format|
   format.any(:json, :pdf) { render json: { success: false, errors: invite_user.errors }, status: :unprocessable_entity }
   format.turbo_stream { turbo_stream_display_error_modal(invite_user.errors)  }
end
```